runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -1924,6 +1941,33 @@ class INTERPRETER_CLASS
 				}
 				break;
 			}
+#if defined(J9VM_OPT_MICROJIT)
+			if(mjitExtra) {
+				if (J9_ARE_NO_BITS_SET(mjitExtra, J9_STARTPC_NOT_TRANSLATED)) {
+					/* Already compiled with MJIT */
+					runMethodCompiled = true;
+				} else if (mjitExtra < 0) {
+					// This method should not be translated by MJIT (already failed), but must keep counting
+				} else if ((IDATA)postCount < (IDATA)mjitExtra) {
+					/* Attempt to compile the method */
+					UDATA *bp = buildMethodFrame(REGISTER_ARGS, _sendMethod, 0);
+					updateVMStruct(REGISTER_ARGS);
+					/* this call cannot change bp as no java code is run */
+					UDATA oldState = pushVMState(REGISTER_ARGS, J9VMSTATE_JIT_CODEGEN);
+					J9JITConfig *jitConfig = _vm->jitConfig;
+					jitConfig->entryPoint(jitConfig, _currentThread, _sendMethod, 0);
+					popVMState(REGISTER_ARGS, oldState);
+					VMStructHasBeenUpdated(REGISTER_ARGS);
+					restoreSpecialStackFrameLeavingArgs(REGISTER_ARGS, bp);
+					/* If the method is now compiled, run it compiled, otherwise run it bytecoded */
+							/* If the method is now compiled, run it compiled, otherwise run it bytecoded */
```

Comment:
Inconsistent indentation.

Reply:
Thanks for catching this, the comment is a duplication so I've removed it.